### ISSUE TITLE
Update 4.2 - 4.3_v3 produces error on all custom modules LBL_NOT_FOUND_QUERY_FIELD_CONDITION

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#5967]
Hi,

I have just re-tested the update from 4.2 to 4.3_v3 and my customised competition (opportunities) module and new custom module with similar fields produce the following error when the menu link is clicked:

LBL_NOT_FOUND_QUERY_FIELD_CONDITION

In the cpanel error log there is no error relating to it - only roundcube errors.
In the cache/logs there are no new errors

This is what it is trying to open (end of the url of if it means anything:
https://www.testsite.com/testcrm/index.php?module=Competition&view=List&mid=218&parent=0

**Comment by kpaulaha** [comment]
I had quite similar problems when updating one from 4.1 to 4.3 with 5 custom modules. At first time it did not succeed, custom modules did not open. Did a rollback and on the second try changed php to 7 before updating  from 4.2 to 4.3 and then on first run did not go to the end, was still 4.2. I did it again and after that everything ok, even custom modules.

**Comment by PercyP** [comment]
@kpaulaha 

thanks for reply. I just tried re-updating but it is exact same issue. It affects one custom module and one existing module I customised - the competition/opportunities module. It seems to be just the menu link to the module that is broken because if I add the following to the url (to add a new record in the module: 
index.php?module=Competition&view=Edit

It allows me to edit and save the record.

But still I cannot open the module - very odd

**Comment by PercyP** [comment]
Hi,

I have found the issue. I have multi-image on system. I basically had to delete all records that had images and then go to the module settings - edit fields and delete all instances of the multi-image field. 

I believe this will be fixed in version 4.4

If there is a possible fix in the meantime it would be good to know what that is

**Comment by mariuszkrzaczkowski** [comment]
All errors that appeared after you uploaded the update package should be described in detail so that we can duplicate the error or find it in the code.
Without this kind of info we won't be able to find and fix it. 
Your errors @PercyP @kpaulaha don't include enough details that would lead us to the solution so we can't really fix them. 
In the "How to report issues" article we described what we need to find the reported errors.

**Comment by kpaulaha** [comment]
Well, in my case upstream timed out with PHP 5.6. After changed to 7, it did go without other changes.

**Comment by mariuszkrzaczkowski** [comment]
Before every update please check the current requirements.

**Comment by kpaulaha** [comment]
Jep, i know that and I have it. It was on our test server with 4 yf installations, different versions. The rest updated ok after knowing the prosedure.

**Comment by PercyP** [comment]
Hi @mariuszkrzaczkowski 

I used the update package from 4.2 to 4.3 v3
I installed it and did not get any error messages on screen. 
It was nothing to do with custom modules in the end. It was to do with records in my system having the MultiImage field and the fact I had existing records with multiImages in them. Other records that had the field (but did not yet have images) worked fine. 
I did not get any errors in the logs - only the error I stated earlier when I attempted to open the module:

LBL_NOT_FOUND_QUERY_FIELD_CONDITION

I deleted all records from my system (test version) and then tried to access the same modules, and they were now accessible. 

So it was just my usual problem with MultiImage field in the end.

DOes that help any more?
